test/test_jobs/test_project_stats.py

```diff
@@ -75,12 +75,12 @@ def test_get_project_jobs(self):
     def test_get_project_jobs_for_non_pro_users(self):
         """Test JOB get project jobs works for non pro users."""
         owner = UserFactory.create(pro=False)
-        oldproject = ProjectFactory.create(owner=owner)
-        created = "2019-10-11T10:00:00"
-        TaskFactory.create(project=oldproject, created=created)
+        old_date = "2019-10-11T10:00:00"
```

Comment:
old task removed that was originally not getting accounted under test with job execution?

Reply:
I created two old projects here. One of the project, I created the a new task, triggering the "updated" column updated in the project. So only one project should be selected. I think that's the expected behavior.

Reply:
Got it. thanks!